### PR TITLE
fix: change default input and textarea shape to rectangle

### PR DESCRIPTION
**Pull request by ychhabra-eightfold**
## SUMMARY:
fix: change default input and textarea shape to rectangle

## CHANGE TYPE:

-   [ ] Bugfix Pull Request
-   [ ] Feature Pull Request

## TEST COVERAGE:

-   [ ] Tests for this change already exist
-   [ ] I have added unittests for this change

### CODE REVIEW

File: src/components/Pagination/Pagination.tsx [hunk 1]
```diff
@@ -315,6 +314,7 @@ export const Pagination: FC<PaginationProps> = React.forwardRef(
                                     <TextInput
                                         ref={inputRef}
                                         classNames={styles.editor}
+                                        shape={TextInputShape.Pill}
```
Comment (jhoward-eightfold) [inline src/components/Pagination/Pagination.tsx:317]: Your commit message and this change seem to be out of sync. changing default to pill?

Comment (ychhabra-eightfold) [reply src/components/Pagination/Pagination.tsx:317]: commit message says changing in input and textarea,

Comment (jhoward-eightfold) [reply src/components/Pagination/Pagination.tsx:317]: Ah okay. Thanks for the clarification... I thought pill shape was the desired look in 2.x styling?

Comment (ychhabra-eightfold) [reply src/components/Pagination/Pagination.tsx:317]: most places where we are using inputs we have been asked to change to rectangle shape by deisgn, so it makes sense to make it the default

Comment (jhoward-eightfold) [reply src/components/Pagination/Pagination.tsx:317]: Sounds good. I'll defer to Dylan on this call.

Comment (dkilgore-eightfold) [reply src/components/Pagination/Pagination.tsx:317]: Will approve for next week's release - need to inform TM, PCS designers and engineering teams about this change. Project X default is Pill, but I see no problem with being discreet about using it.

**Review by dkilgore-eightfold — APPROVED** [review]

**Pull request merged**